finetuner-workflow/finetune-workflow.yaml

```diff
@@ -310,53 +336,90 @@ spec:
             value: "{{workflow.parameters.trainer_gpus}}"
         when: "{{workflow.parameters.inference_only}} == false"
 
-    - - name: inference-service
+    - - name: vllm-serialization
+        template: model-vllm-serialization
+        arguments:
+          parameters:
+            - name: model_path
+              value: "/{{workflow.parameters.pvc}}/finetunes/results-{{workflow.parameters.run_name}}/final"
+        when: "{{workflow.parameters.run_inference}} == true"
+
+    - - name: vllm-inference-service
         template: model-inference-service
         arguments:
           parameters:
+            - name: model_uri
+              value: "vllm/{{workflow.parameters.pvc}}/finetunes/results-{{workflow.parameters.run_name}}/final/vllm/model.tensors"
```

Comment:
flagging to ensure consistency with https://github.com/coreweave/kubernetes-cloud/pull/376/files#r1695728991

Reply:
This was discussed a while ago, the final output path being pretty wonky was a consequence of the serialization script used. However, I think the finetuner should have it's own vLLM serialization script which by the looks of things doesn't seem hard to do.